ROMFS/px4fmu_common/init.d/rc.sensors

```diff
@@ -146,39 +144,47 @@ then
 	# External I2C bus
 	hmc5883 -C -T -X start
 
-	lis3mdl -R 2 start
-
-	# Internal SPI bus is rotated 90 deg yaw
-	hmc5883 -C -T -S -R 2 start
-
-	# Internal SPI bus ICM-20608-G is rotated 90 deg yaw
-	mpu6000 -R 2 -T 20608 start
-
-	# Internal SPI bus ICM-20602-G is rotated 90 deg yaw
-	mpu6000 -R 2 -T 20602 start
+	if hmc5883 -C -T -S -R 2 start
+	then
+		# hmc5883 internal SPI bus is rotated 90 deg yaw
+	else
+		lis3mdl -R 2 start
+	fi
 
-	# Start either MPU9250 or BMI160. They are both connected to the same SPI bus and use the same
+	# Start either ICM2060X or BMI055. They are both connected to the same SPI bus and use the same
```

Comment:
I would think we could check one Bosch sensor first. If it is present do the only Bosch path.  I do not have the HW to validate this, do you?

Reply:
I only have a couple standard pixracers. Looking at the code again, the bmi055 gyro chip select is actually shared with the hmc, although grouping it like this shouldn't matter.

Also the external i2c bmm150 isn't actually pixracer specific, but I think it was thrown in at the same time for development/testing.

At this point it's largely cosmetic for keeping the console clean during boot (and slightly faster).

Reply:
@dagar is [this](https://github.com/PX4/Firmware/blob/master/src/drivers/boards/px4fmu-v4/board_config.h#L66-L72) a complete picture of the shared usage or is is incomplete?